### PR TITLE
Add Zabbix project to vagrant_infrastructure

### PR DESCRIPTION
**Pull request by codybennett**
Add Zabbix Role requirement

Motivation and Context
----------------------

Testing
-------------------------

### CODE REVIEW

**Comment by codybennett** [comment]
Added zabbix.vagrant.localdomain to host_vars
Created Stand-alone zabbix box in vagrant.yml

File: projects/zabbix/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Vagrant Infrastructure - Generic
```
Comment (lo5an) [inline projects/zabbix/README.md]: It'd be awesome to have more docs. Maybe more on the setup?

File: projects/zabbix/host_vars/dev.vagrant.localdomain/main.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (lo5an) [inline projects/zabbix/host_vars/dev.vagrant.localdomain/main.yml]: Does a `dev` box get provisioned? If not, we should probably delete the vars folder from host_vars.

File: projects/zabbix/inventory.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```
Comment (lo5an) [inline projects/zabbix/inventory.py]: This script will get overwritten by the bootstrap process. I suspect it is not needed.

File: projects/zabbix/playbooks/vagrant.retry [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apigate.vagrant.localdomain
```
Comment (lo5an) [inline projects/zabbix/playbooks/vagrant.retry]: this a junk file! we should delete it, and maybe add a pattern for `vagrant.retry` to `.gitignore`.

**Comment by lo5an** [comment]
This work has been superseded.

**Pull request closed without merging**